snap/snapcraft.yaml

```diff
@@ -40,6 +40,9 @@ parts:
     override-build: |
       cp /etc/ssl/certs/java/cacerts $SNAPCRAFT_PART_INSTALL/usr/lib/jvm/java-17-openjdk-$SNAPCRAFT_TARGET_ARCH/lib/security/cacerts
       cp /etc/ssl/certs/java/cacerts $SNAPCRAFT_PART_INSTALL/etc/ssl/certs/java/cacerts
+    override-prime: |
+      snapcraftctl prime
```

Comment:
Is it worth using the craftctl tool instead? From the docs,
>snapcraftctl is deprecated in favour of craftctl from core22 onwards. Please refer to [Using craftctl tool](https://snapcraft.io/docs/using-craftctl) for more information about the craftctl tool.

Reply:
Seems we're using core20 here so maybe something for later if/when we switch to core22.